### PR TITLE
New profile linking

### PR DESCRIPTION
**Pull request by dannypernik**
Everyone gets directed to profile.html, and admins have a link to admin-profile.

### CODE REVIEW

File: openCurrents/templates/admin-profile.html [hunk 1]
```diff
@@ -12,64 +12,63 @@
...
+              <a href="{% url 'openCurrents:invite-volunteers' %}" class="button secondary tiny round">Invite volunteers</a>
+            <!-- </span> -->
+          </div>
 
-          <!--<div class="medium-5 medium-offset-1 columns">
-            <img id="profile-img" src="{% static 'img/smiley.jpg' %}" class="round" /> -->
-          <h3 class="title-sub">{{ user.first_name }} {{ user.last_name }}</h3>
-          <h5 class="three-halves-margin-bottom">{{ org_name }}</h5>
+          <div class="one-margin-top right medium-4 large-3 columns end">
+            <h6 class="no-margin-bottom two-line-height">My hours tracked: [my]</h6>
```
Comment (nickolashe) [inline openCurrents/templates/admin-profile.html]: @dannypernik what does [my] stand for?

Comment (dannypernik) [inline openCurrents/templates/admin-profile.html]: Admin's hours tracked

Comment (nickolashe) [inline openCurrents/templates/admin-profile.html]: it shows on the screen as `[my]`, is this what you intended?

Comment (nickolashe) [inline openCurrents/templates/admin-profile.html]: [my] refers to hours tracked by admin only

**Pull request merged**